src/Transports/MassTransit.EventHubIntegration/EventHubIntegration/EventHubProducerSendTransportContext.cs

```diff
@@ -175,7 +175,7 @@ public Task Send(IPipe<ProducerContext> pipe, CancellationToken cancellationToke
         }
 
         public override string EntityName => _endpointAddress.EventHubName;
-        public override string ActivitySystem => EventHubEndpointAddress.PathPrefix;
+        public override string ActivitySystem => "eventhubs";
```

Comment:
same here, can we turn off mass-transit instrumentation and use Azure SDK native one? 
Here're the docs on existing instrumentation https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md#distributed-tracing